src/robotlibcore.py

```diff
@@ -141,6 +131,29 @@ def get_keyword_documentation(self, name):
             doc = '{}\n\n{}'.format(doc, tags) if doc else tags
         return doc
 
+    def get_keyword_types(self, name):
+        kw = self.keywords[name]
+        if hasattr(kw, 'robot_types'):
+            robot_types = getattr(kw, 'robot_types')
+            if robot_types:
+                return robot_types
+        default_types = self._get_default_types(kw)
+        if default_types:
+            return default_types
+        return {} if PY2 else kw.__annotations__
```

Comment:
What `__annotations__` contain can be different depending on Python version (mostly how `typing` behaves) and depending on is `from __future__ import annotations` enabled (see [PEP 563](https://www.python.org/dev/peps/pep-0563/)). All this ought to be handled by Robot, but it's good to add tests for various usages.

Reply:
Will test. Should we setup a CI for this project? It is not that much of work and would be easy the work for different Python versions.